src/frontend/src/expr/agg_call.rs

```diff
@@ -68,32 +91,25 @@ impl AggCall {
                 DataType::Float32 => DataType::Float32,
                 DataType::Float64 => DataType::Float64,
                 DataType::Interval => DataType::Interval,
-                _ => return None,
+                _ => return unsupported(),
```

Comment:
avg ( smallint ) → numeric
avg ( integer ) → numeric
avg ( bigint ) → numeric
avg ( numeric ) → numeric
avg ( real ) → double precision
avg ( double precision ) → double precision
avg ( interval ) → interval
sum ( smallint ) → bigint
sum ( integer ) → bigint
sum ( bigint ) → numeric
sum ( numeric ) → numeric
sum ( real ) → real
sum ( double precision ) → double precision
sum ( interval ) → interval
sum ( money ) → money

https://www.postgresql.org/docs/current/functions-aggregate.html

```suggestion
                _ => return invalid(),
```

We have actually supported the full valid type list, so maybe we should use invalid?

Reply:
Good catch! Will fix.